src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -121,8 +123,8 @@ public static boolean RequestHTTPS(
         // json has odd characters at the front, trim them to make valid json
         int jsonStartIndex = fileStr.indexOf("{");
 
-        int notFound  = -1;
-        if(jsonStartIndex == notFound){
+        int notFound = -1;
+        if (jsonStartIndex == notFound) {

```

Comment:
Same comment as above -- the code added in the previous commit appears to be formatted in this commit. Please move to the original commit (no need to commit new code that already needs formatting, just format from the start).